airbyte-integrations/connectors/source-paystack/metadata.yaml

```
@@ -5,12 +5,13 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 193bdcb8-1dd9-48d1-aade-91cadfd74f9b
-  dockerImageTag: 0.1.4
+  dockerImageTag: 0.1.5
```

```suggestion
  dockerImageTag: 0.2
```

I get that it might not be breaking, but it's a big rewrite. Gut instinct: call it at least minor bump.